src/TipBox.vue

```diff
@@ -185,11 +185,14 @@
     .icon-wrapper {
         display: flex;
         margin-right: .5em;
-        width: 22px;
-        height: 22px;
     }
 
     .contents {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-content: center;
+        font-size: medium;
```

Comment:
I believe we only need `display` and `align-content`.

Reply:
I tried it without the `flex-direction: column`, and the `justify-content: center`. Here's what it looks like for large paragraphs - 
<img width="809" alt="Screenshot 2020-01-24 at 5 16 13 PM" src="https://user-images.githubusercontent.com/21968718/73057751-411f2780-3ecd-11ea-9082-099b5829affd.png">

Here's what it looks like without `justify-content: center` (but with `flex-direction: column`). While it does work for long paragraphs, single-line content is not properly aligned -
<img width="814" alt="Screenshot 2020-01-24 at 5 19 51 PM" src="https://user-images.githubusercontent.com/21968718/73058046-c276ba00-3ecd-11ea-871c-1426e742a4e8.png">

Reply:
That appears to be because `<slot></slot>` is directly in `contents`, together with `heading` and `dismiss-button`.

I don't think it is appropriate to always centre the children in `contents` though, particularly the `heading`. Did we try `extra-large` (maybe extremely-large) with `heading` and/or `dismissable`?

Reply:
Sorry for the late reply. I've been trying out different things to make this work. You're right, the `heading` and `dismiss-button` don't behave well when I center-align the contents. Center-aligning the contents is much harder than I thought it would be :sweat_smile:

I've tried wrapping the `<slot>` in a `<div>` and `<span>`, but that doesn't work well. The next thing I tried was moving the `heading` and `button` outside the 'content's `div` and make it a 3 column container (one for the icon, one for text, one for heading and button). The problem here is that the `heading` messes with the text column if it is too long. 
So, the latest thing I'm working on is separating out the `heading` into a `div` that would be placed on top of the container, effectively making it a 2-rowed container. The 2nd row will be a 3 column container, with the 3rd column only for the `dismiss-button`. 

Are there any better ways to deal with this?

Reply:
I would say not to vertically center-align the contents.

Reply:
Alright 👍